vpn-store/schemas/com.duckduckgo.mobile.android.vpn.store.VpnDatabase/20.json

```diff
@@ -640,7 +646,7 @@
     "views": [],
     "setupQueries": [
       "CREATE TABLE IF NOT EXISTS room_master_table (id INTEGER PRIMARY KEY,identity_hash TEXT)",
-      "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, 'd683be5ff8d34422f26e050b3522918d')"
+      "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, 'dc6e3395763a09e6953c47ec53d5566e')"
```

Comment:
This file, ie. `20.json` should not change, the `21.json` is the newly created one